svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -155,54 +129,6 @@ public function addActionsAndFilters()
         // $this->guildLog('Adding filter ' . "{$prefix}plugin_action_links_{$plugin_file}");
         add_filter("{$prefix}plugin_action_links_{$plugin_file}", array(&$this, 'onActionLinks'));
 
-        $reserveFilmstripSpace = $this->getOption('ReserveFilmstrip', 'None');
```

Comment:
Wasn't expecting to see this much code removal, but if we don't need it, then good riddance!